WalletWasabi.Backend/Global.cs

```diff
@@ -113,6 +113,7 @@ public async Task InitializeAsync(CoordinatorRoundConfig roundConfig, Cancellati
 				}
 
 				HttpClient.BaseAddress = url;
+				HttpClient.Timeout = TimeSpan.FromMinutes(2);
```

Comment:
I think we should keep the default timeout. There must be a reason for 100 seconds. Can we retry in that case?

Reply:
I agree with David here. Timeouts are a solution, not a problem. I would suggest you read about the retry policies introduced in net5 and avoid implementing custom code for that, if possible.

Reply:
There is already a retry logic in the coinverifier, that needs to be adjusted.

Reply:
Added a small delay between the retries. Plus wrapped the `SendAsync` with a try-catch block.